addons/account/tests/test_account_invoice_report.py

```diff
@@ -55,6 +56,12 @@ def setUpClass(cls):
                         'quantity': 1,
                         'price_unit': 1200,
                     }),
+                    Command.create({
```

Comment:
Same here, why not changing the others (0, None, ...
Either switch them all or don't switch the system at all